sorc/build_ufs.sh

```diff
@@ -7,12 +7,13 @@ cwd=$(pwd)
 APP="S2SWA"
 CCPP_SUITES="FV3_GFS_v17_p8_ugwpv1,FV3_GFS_v17_coupled_p8_ugwpv1"  # TODO: does the g-w need to build with all these CCPP_SUITES?
 
-while getopts ":da:j:v" option; do
+while getopts ":da:j:vw" option; do
   case "${option}" in
     d) BUILD_TYPE="DEBUG";;
     a) APP="${OPTARG}";;
     j) BUILD_JOBS="${OPTARG}";;
     v) export BUILD_VERBOSE="YES";;
+    w) export PDLIB="ON";; 
```

Comment:
I don't think this needs to be exported, since it is just turned into a cmake argument in this script
```suggestion
    w) PDLIB="ON";; 
```

Reply:
I will test this change before committing it to the branch.